nextflow_schema.json

```diff
@@ -44,6 +44,24 @@
                 }
             }
         },
+        "main_options": {
```

Comment:
Just a suggestion, perhaps we could call this "analysis_options"?

Reply:
@ramprasadn Sure! Do you think future options will be better suited under 'analysis_options' vs. 'main_options'?

Reply:
done!

Reply:
Ah sorry! I somehow missed my notification. Yes, while I am not sure how this might turn out in the future, I just thought it'd be nice to have something more descriptive like, we already have in MIP.